Mathlib/Analysis/SpecialFunctions/Integrals/LogTrigonometric.lean

```diff
@@ -0,0 +1,78 @@
+/-
+Copyright (c) 2025 Stefan Kebekus. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Stefan Kebekus
+-/
+import Mathlib.Analysis.SpecialFunctions.Integrability.LogMeromorphic
+
+/-!
+# Integral of `log ∘ sin`
+
+This file computes special values of the integral of `log ∘ sin`. Given that the indefinite integral
+involves the dilogarithm, this can be seen as computing special values of `Li₂`.
+-/
+
+open Filter Interval Real
+
+/--
+Helper lemma for `integral_log_sin_zero_pi_div_two`: The integral of `log ∘ sin` on `0 … π` is
+double the integral on `0 … π/2`.
+-/
+lemma integral_log_sin_zero_pi_eq_two_mul_integral_log_sin_zero_pi_div_two :
```

Comment:
in my opinion this lemma is strictly less useful than the two below; perhaps it makes sense to make this one private?